jobs/start-regulering/src/test/java/ApplicationKtTest.kt

```diff
@@ -0,0 +1,12 @@
+import no.nav.etterlatte.createRecord
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.Month
+
+class ApplicationKtTest {
+    @Test
+    fun `lager record`() {
+        val record = createRecord(LocalDate.of(2024, Month.MAY, 21))
```

Comment:
Hva er verdien her?

Reply:
Ja, denne "testen" burde kanskje fjernes

Reply:
Ups, ja, hadde ikkje tenkt å committe denne, kun for å forenkle testing.